recipes/mlst/meta.yaml

```diff
@@ -1,13 +1,14 @@
 {% set name = "mlst" %}
-{% set version = "2.16.4" %}
-{% set sha256 = "e9a3de71450410f4ee6a19f0aef6da55a0a0dd1def8d90753fb230432306e662" %}
+{% set version = "2.17.6" %}
+{% set sha256 = "223c6b4751bc4fb7fe9d41f56e75c90235b1eba022f987abc1d75ae51fc50819" %}
+{% set user = "tseemann" %}
 
 package:
   name: '{{ name }}'
   version: '{{ version }}'
 
 source:
-  url: https://github.com/tseemann/{{ name }}/archive/v{{ version }}.tar.gz
+  url: https://github.com/{{ user }}/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
You really like these variables, aren't you :)

Reply:
I :heart: them.  Trust me, it will save a lot of time when I die and pass on my repos to you @bgruening :)

Reply:
:rofl: